package.json

```diff
@@ -13,8 +13,8 @@
   },
   "peerDependencies": {
     "esri-leaflet": ">2.3.0",
-    "leaflet": "^1.5.0",
-    "maplibre-gl": "^2.2.1"
+    "leaflet": "^1.9.4",
+    "maplibre-gl": "^2.4.0"
```

Comment:
If I recall correctly, peer dependencies are saying "you need at least this version to work with esri-leaflet-vector" ... so I think we want to **leave these as-is** as long as esri-leaflet-vector still works with leaflet@1.5.0 and maplibre-gl@2.2.1 .... I could be wrong on this though - npm dependencies is a bit of a confusing topic ... @patrickarlt could you please confirm?